src/components/MdDatepicker/MdDatepicker.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <md-field class="md-datepicker">
+  <md-field :class="['md-datepicker', { 'md-native': !this.mdOverrideNative }]">
```

Comment:
I think this should be the opposite statement with `!isFirefox || this.mdOverrideNative`.

There could be a computed called `native`.

check here:
https://github.com/vuematerial/vue-material/blob/6a719b5c2421d2efadb3bc522693e438a5a01066/src/components/MdDatepicker/MdDatepicker.vue#L85